DuckDuckGo/Bookmarks/Extensions/Bookmarks+Tab.swift

```diff
@@ -28,6 +28,13 @@ extension Tab {
         }
     }
 
+    @MainActor
+    static func withContentOfBookmark(bookmarks: [Bookmark], burnerMode: BurnerMode) -> [Tab] {
```

Comment:
Please rename according to the Swift naming convention: `with(contentsOf bookmarks: [Bookmark])`
And I think it‘s better to make it `extension [Tab]` with `init(contentsOf bookmarks: [Bookmark])` so the called object convenience initializer matches the returned result.